src/coreclr/jit/compiler.h

```diff
@@ -9553,7 +9553,8 @@ class Compiler
     //
     bool canUseEvexEncoding() const
     {
-        return compOpportunisticallyDependsOn(InstructionSet_AVX512F);
+        return (compOpportunisticallyDependsOn(InstructionSet_AVX512F) ||
+                compOpportunisticallyDependsOn(InstructionSet_AVX10v1));
```

Comment:
I wonder if we want to order AVX10v1 first and/or if we want to cache this to avoid repeated lookups, since which is primary will change over time.

Reply:
I will move the Avx10v1 check first as of now. Have we cached any other similar checks? not the ISA ones but any similar checks previously? I was thinking we can cache it later once AVX10v1 becomes a commonly available ISA.